src/utilities/data-formatter.ts

```diff
@@ -291,4 +300,4 @@ function defaultRNNFormatter(data) {
   return result;
 }
 
-module.exports = { DataFormatter, defaultRNNFormatter };
+export default DataFormatter;
```

Comment:
We've not been using default exports.  Can we just use named ones?

Reply:
an import in  src/index.ts:
`import DataFormatter from './utilities/data-formatter';`
is assuming there is a default export, or it needs to be converted to 
`import * as DataFormatter from './utilities/data-formatter'; `

The naming makes me think the first option is what was intended, but It is a change, should I change it to the second option?

Reply:
Go ahead and use named exports, and change wherever there are imports.

Reply:
Done